crosstests/common/src/main/kotlin/build/buf/connect/crosstest/ssl/SSL.kt

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package build.buf.connect.crosstest.ssl
+package com.connectrpc.crosstest.ssl
```

Comment:
Not required for this change, but we may want to consider changing the top-level package to `com.connectrpc.kotlin.*`? That way if there is ever another implementation targeting Java or another JVM language, it would have a place to live.

Reply:
that makes sense to me! but ya maybe we do that as a separate change